project/SparkSnowflakeBuild.scala

```diff
@@ -72,6 +72,7 @@ object SparkSnowflakeBuild extends Build {
         "com.amazonaws" % "aws-java-sdk-core" % "1.10.22" % "provided" exclude("com.fasterxml.jackson.core", "jackson-databind"),
         "com.amazonaws" % "aws-java-sdk-s3" % "1.10.22" % "provided" exclude("com.fasterxml.jackson.core", "jackson-databind"),
         "com.amazonaws" % "aws-java-sdk-sts" % "1.10.22" % "test" exclude("com.fasterxml.jackson.core", "jackson-databind"),
+        "com.microsoft.azure" % "azure-storage" % "5.0.0" % "provided" exclude("com.fasterxml.jackson.core", "jackson-databind"),
```

Comment:
Here. In particular, we assume azure-storage is already installed in Spark.

Reply:
I am assuming this assumption is the same for S3? If users want to use Azure, they need to make sure Azure libraries are included in their classpath, so is S3 right?